### ISSUE TITLE
Doesn't match simple evil case.

### ISSUE THREAD

**Issue by RealAlphabet** [gagyibenedek/ReDoS-checker#2]
Evil regex : 
`(a+?)(b+)`

For example, with this as input : 
`(a times 1000)zab`

The number of steps increases exponentially.

**Comment by natrajms** [comment]
Yes. This has a bug. I tried the evil regex and found that it is not working.
^([a-zA-Z]+[,.]?[ ]?|[a-zA-Z]+['-]?)+$
